pkgs/top-level/all-packages.nix

```diff
@@ -4630,6 +4630,8 @@ in
 
   latexrun = callPackage ../tools/typesetting/tex/latexrun { };
 
+  pplatex = callPackage ../tools/typesetting/tex/pplatex { };
```

Comment:
Seems like this place is for packages starting with a letter "L"

Reply:
I've noticed that not everybody follows the ordering and hence deliberately placed it where I figured it may make more sense. Happy to change if you would like

Reply:
Yes, there are some problems with ordering, let's not introduce more.

If you want to keep it in the "### DEVELOPMENT" section, then it makes sense to put it after "ppl = callPackage ../development/libraries/ppl { };".